client/my-sites/subscribers/components/subscriber-launchpad/subscriber-launchpad.tsx

```diff
@@ -1,6 +1,5 @@
 import { CircularProgressBar } from '@automattic/components';
 import { Launchpad } from '@automattic/launchpad';
-import i18n, { useTranslate } from 'i18n-calypso';
```

Comment:
We still need to keep `useTranslate` imported as we need it for translations in the same file. 🙂 

e.g.:
```
const translate = useTranslate();

//...

translate( 'No subscribers yet?' );
```